src/module/server-group/ServerGroup.ts

```diff
@@ -34,4 +39,65 @@ module.config(($stateProvider: ng.ui.IStateProvider) => {
         }]
     }
   });
+  $stateProvider.state("server-group.nodes", {
+    url: "/nodes",
+    templateUrl: "module/server-group/nodes/view/server-group.html"
+  });
+
+  $stateProvider.state("server-group.endpoints", {
+    url: "/endpoints",
+    controller: EndpointsCtrl,
+    controllerAs: "ctrl",
+    templateUrl: "module/server-group/endpoints/view/endpoints.html",
+    resolve: {
+      endpoints: ["endpointService", "serverGroup",
+        (endpointService: EndpointService, serverGroup: IServerGroup) => {
+          return endpointService.getAllClusterEndpoints(serverGroup);
+        }],
+    }
+  });
+
+  $stateProvider.state("edit-endpoint-config", {
+    parent: "root",
+    url: "/server-groups/:serverGroup/endpoints/:endpointType/:endpointName/config",
```

Comment:
This should be different once we have exposed an Endpoint module, but when inheriting from root you should start with "/" otherwise we get "http://localhost:3000/#//server-groups/cluster/endpoints/hotrod-connector/hotrod-connector/config"